modular_zubbers/code/modules/emotes/emotes.dm

```diff
@@ -109,3 +109,20 @@
 			return 'modular_zubbers/code/modules/emotes/sound/voice/male_sigh_exasperated.ogg'
 		return 'modular_zubbers/code/modules/emotes/sound/voice/female_sigh_exasperated.ogg'
 	return
+
+/datum/emote/living/tail_thump
```

Comment:
Needs a can_run_emote proc for human mobs so it can't be run without a tail. Similar to how `/datum/emote/living/carbon/human/wag` does it but you just need to check if the organ exists

Reply:
Tail check added. It compiles and worked when testing between a character with and without a tail. Hooray.